code/game/objects/structures/beds_chairs/alien_nest.dm

```diff
@@ -50,9 +50,7 @@
 		add_fingerprint(user)
 
 /obj/structure/bed/nest/user_buckle_mob(mob/living/M, mob/living/user)
-	if ( !ismob(M) || (get_dist(src, user) > 1) || (M.loc != src.loc) || user.restrained() || user.stat || M.buckled || istype(user,
-
-/mob/living/silicon/pai) )
```

Comment:
Why was the code checking for pai specifically? Noone knows. In any case, pais don't pass the getorgan check below.
Also, I changed restrained+stat check to use incapacitated() so it also check for stunned/weakened.